packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -42,17 +44,21 @@ export const PeoplePaneContent = (props: {
   alternateCallerId?: string;
 }): JSX.Element => {
   const { inviteLink, onFetchParticipantMenuItems, setDrawerMenuItems, strings, onRemoveParticipant } = props;
-
+  /* @conditional-compile-remove(rooms) */
+  const isRemovable = _usePermissions().removeParticipantButton;
   const participantListDefaultProps = usePropsFor(ParticipantList);
-
   const setDrawerMenuItemsForParticipant: (participant?: ParticipantListParticipant) => void = useMemo(() => {
     return (participant?: ParticipantListParticipant) => {
       if (participant) {
+        let disableRemovebutton = !participant.isRemovable;
+        /* @conditional-compile-remove(rooms) */
+        disableRemovebutton = !isRemovable || disableRemovebutton;
```

Comment:
Specially if you introduce the trampoline function as requested above, you can just call it repeatedly and drop this `let`: `const disableRemoveButton = !hasRemoveParticipantsPermissionTrampoline() || !participant.isRemovable`

Reply:
done